src/plugins/intel_gpu/src/graph/include/fully_connected_inst.h

```diff
@@ -47,6 +47,7 @@ class typed_primitive_inst<fully_connected> : public typed_primitive_inst_base<f
     template<typename ShapeType>
     static std::vector<layout> calc_output_layouts(fully_connected_node const& /*node*/, const kernel_impl_params& impl_param);
     static layout calc_output_layout(fully_connected_node const& node, kernel_impl_params const& impl_param);
+    static kernel_impl_params get_fake_aligned_params(kernel_impl_params const& orig_impl_param);
```

Comment:
Also, the feature itself may be useful for static cases too. So please check if current solution can be further extended to static models

Reply:
Yes, it might be. I will create another task for that in the normal optimization backlog. I think we need to find a network that can be improved by this.